site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -245,16 +245,31 @@ After the deployment is completely removed, it will have a "Destroyed" state on
 
 == Modifying Deployments
 
-If you are on the High-Availability plan and want a custom domain name such as `auth.example.com`, open a https://account.fusionauth.io/account/support/[support ticket]. This feature requires TLS certificate setup and therefore coordination with you.
-
 Open a https://account.fusionauth.io/account/support/[support ticket] to change any of the following attributes of your deployment:
 
 * the tier
 * the region or geographic location
 * the size
 * the hostname
 
-Such changes are typically handled in 1-2 business days. 
+Such changes are typically handled in 1-2 business days.
+
+=== Data Integrity
+
+A common question is -- will I lose any data when migrating between tiers, region, size, or hostname?
+
+When performing any changes which require modifying cloud infrastructure, the cloud database is backed up, and then reconnected to the new infrastructure.
+
+No data loss should occur and your configuration will remain.  Any perceived loss could indicate the need to rebuild the elastic index.  Please see link:/docs/v1/tech/core-concepts/search/#reindexing-elasticsearch[here] for additional details.
```

Comment:
> ... will remain intact. 

or 

> ... will be preserved.

Reply:
No big deal, but in general I think we are using a single space in between sentences?

Reply:
> Any perceived loss could indicate the need to rebuild the elastic index. Please see link:/docs/v1/tech/core-concepts/search/#reindexing-elasticsearch[here] for additional details.

Do we want to mention this? If we perform the migration - we should either be rebuilding the index for the customer or it would be part of the automated migration steps.

Reply:
Agree - we could add this to the migration steps (if not already there).  I can remove it.